pithos/plugins/notification_icon.py

```diff
@@ -112,6 +79,18 @@ def check_for_plasma():
                 None
             )
 
+        def on_StatusNotifierWatcher_finish(source, result, data):
+            try:
+                StatusNotifierWatcher = Gio.DBusProxy.new_finish(result)
+            except GLib.Error as e:
+                logging.debug(e)
+                check_for_x()
+            else:
+                if StatusNotifierWatcher.get_name_owner():
+                    init_appindicator()
+                else:
+                    check_for_plasma()
```

Comment:
Also this might be beyond reasonable, but appindicator is trash, I just built it for the flatpak. Might make more sense to rewrite it in a few lines of python...